### PR TITLE
Update docs for RDMO 2.0

### PR DESCRIPTION
**Pull request by jochenklar**
Update the docs for RDMO and the new `rdmo-app`.

Still missing:

- [x] update of the management part
- [x] move the `/docs` of the rdmo repo into `/development`

I will work on this tomorrow.

### CODE REVIEW

File: docs/advanced/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+For distributions that use SELinux (e.g. CentOS, RHEL, Fedora), the following commands can be used to configure SELinux:
+
+```bash
+sudo semanage fcontext -a -t httpd_sys_content_t "/srv/rdmo/rdmo-app(/.*)?"
+sudo semanage fcontext -a -t httpd_sys_rw_content_t "/srv/rdmo/rdmo-app/static_root/CACHE(/.*)?"
+sudo semanage fcontext -a -t httpd_sys_rw_content_t "/srv/rdmo/rdmo-app/log(/.*)?"
+sudo semanage fcontext -a -t httpd_sys_script_exec_t -f f "/srv/rdmo/rdmo-app/env(/.*)?/.+\.so(\.[^/]*)*"
+sudo restorecon -R -v /srv/rdmo
+sudo setsebool -P httpd_can_network_connect=1
+# Run this only if using memcached
```
Comment (afuetterer) [inline docs/advanced/index.md:14]: Remove?
see discussion about memcached in https://github.com/rdmorganiser/rdmo/pull/649

File: docs/deployment/gunicorn.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
+# Gunicorn
+
+As mentioned several times, you should create a dedicated user for RDMO. All steps for the installation, which do not need root access, should be done using this user. Here we assume this user is called `rdmo` and it's home is `/srv/rdmo` and therefore your `rdmo-app` is located in `/srv/rdmo/rdmo-app`.
+
+First install gunicorn inside your virtual environment:
+
+```bash
+pip install -r requirements/gunicorn.txt
```
Comment (afuetterer) [inline docs/deployment/gunicorn.md:8]: should be `pip install rdmo[gunicorn]`

File: docs/deployment/gunicorn.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+## Apache2 as reverse proxy
+
+Apache can be installed on Debian or Ubuntu using:
+
+```bash
+sudo apt install apache2
+```
+
+
+Edit the Nginx configuration as follows (again with root/sudo permissions):
```
Comment (afuetterer) [inline docs/deployment/gunicorn.md:127]: should be Apache configuration

File: docs/deployment/gunicorn.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    </Location>
+
+    Alias /static /srv/rdmo/rdmo-app/static_root/
+    <Directory /srv/rdmo/rdmo-app/static_root/>
+        Require all granted
+    </Directory>
+</VirtualHost>
+```
+
+Restart and enable Nginx:
```
Comment (afuetterer) [inline docs/deployment/gunicorn.md:149]: apache

File: docs/installation/index.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
 # Installation
 
-For demonstration, development or testing purposes, RDMO can be installed on Linux, Windows and macOS. If you, however, to set up a production environment, serving RDMO over a Network or the Internet, we suggest that you use a recent Linux distribution, namely CentOS, Debian, or Ubuntu LTS.
+For demonstration, development or testing purposes, RDMO can be installed on Linux, Windows and macOS. If you, however, to set up a production environment, serving RDMO over a Network or the Internet, we suggest that you use a recent Linux distribution. We suggest to use Debian or Ubuntu LTS.
 
-The code is mainly written in Python and should work with a Python higher than 3.6.
+The code is mainly written in Python and works with a version higher than **Python 3.8**.
```
Comment (afuetterer) [inline docs/installation/index.md:5]: When you say higher than in prose it sounds like
Python > 3.8 (not inclusive).

I would recommend:

The code is mainly written in Python and works with version **Python 3.8** or higher.

**Comment by MyPyDavid** [comment]
I'll include the comments from @afuetterer  and make my review now.

Don't you want to have this (https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx) included in some of the code blocks?

**Comment by afuetterer** [comment]
Will you update `configuration/databases.md` as well?

It still states to do `pip install -r requirements/postgres.txt`

Also this line should be changed to `'ENGINE': 'django.db.backends.postgresql',`
https://github.com/rdmorganiser/rdmo-docs-en/blob/ac00a6397588ee9c95073143fa2a18e6b8ce38e9/docs/configuration/databases.md?plain=1#L31

**Comment by MyPyDavid** [comment]
thanks! Yes I am going through all of the docs, rewriting some parts, updating links and refs to 4.2.

**Comment by MyPyDavid** [comment]
I've added the tabbed code blocks with https://github.com/executablebooks/sphinx-tabs

**Pull request merged**